bioptim/examples/muscle_driven_ocp/muscle_activations_tracker.py

```diff
@@ -98,7 +98,7 @@ def generate_data(
     def dyn_interface(t, x, u):
         if use_residual_torque:
             u = np.concatenate([np.zeros(n_tau), u])
-        return np.array(dynamics_func(x, u, [])).squeeze()
+        return np.array(dynamics_func(x, u, [])[:, 0]).squeeze()
```

Comment:
I am not sure why it is not just `[0]`?

Reply:
nope, I need all the states from the first column. :+1: